### DISCUSSION TITLE
Table permissions and relationships are reverted after some time automatically.

### DISCUSSION THREAD

**Discussion by versa-dev** [hasura/graphql-engine#8567] [General] [answered]
I have created new table, and update the permissions and relationships with other tables.
And confirmed it is working well as I expected. But after couple of hours without touching the table and any requests, the permissions are reverted, and I should add the relationships again. This problem is happening every day I am trying to start working.

**Comment by BenoitRanque** [comment, marked as answer]
Do you happen to be using a cli-migrations version of the hasura image?
These will apply migrations and metadata automatically on startup.

If you do not export your metadata, and restart your computer, your old metadata will be applied again and replace all the new work.
There are a few ways to avoid this, the easiest is to run the [cli console](https://hasura.io/docs/latest/graphql/core/hasura-cli/hasura_console/) which automatically exports your metadata each time you make a change.
